louiCoder/Node-WebScrapper
# Set up server #15

#### What does this PR do?
- creates a server for handling requests that will be made in the application.

#### Any more information to look out for when testing this PR
To test this:
- Go to **server.js** and find `scrapUrl` function and replace the argument with a string **url** of your choice.
- Make sure to run **npm install** to install all dependencies.
- With the above completed, now start the server with the command **_npm start_**
